src/test/java/com/suse/saltstack/netapi/calls/modules/TestTest.java

```diff
@@ -0,0 +1,48 @@
+package com.suse.saltstack.netapi.calls.modules;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import java.io.InputStream;
+
+import org.junit.Test;
+
+import com.google.gson.reflect.TypeToken;
+import com.suse.saltstack.netapi.calls.modules.Test.VersionInformation;
+import com.suse.saltstack.netapi.parser.JsonParser;
+
+/**
+ * Test unit tests.
+ */
+public class TestTest {
+
+    @Test
+    public void testTestPing() {
+        TypeToken<Boolean> type =
+                com.suse.saltstack.netapi.calls.modules.Test.ping().getReturnType();
```

Comment:
I think we should use the fully qualified name for the `@Test` since nobody looks at them anyway so it does not matter if they are longer but making the test code shorter should make it more readable.